_includes/markdown/Maintaining.md

```diff
@@ -54,3 +54,14 @@ In the scenario where your organization uses GitHub for code and publishes code
 <h2 id="version-control" class="anchor-heading">Version Control {% include Util/link_anchor anchor="version-control" %}</h2>
 
 In general we follow the [Version Control guidelines from the 10up Engineering Best Practices](https://10up.github.io/Engineering-Best-Practices/version-control/#top) including the approaches for [structure](https://10up.github.io/Engineering-Best-Practices/version-control/#structure-package-management) and [workflows](https://10up.github.io/Engineering-Best-Practices/version-control/#workflows).
+
+<h2 id="version-control" class="anchor-heading">WordPress and PHP Minimums {% include Util/link_anchor anchor="wp-php-mins" %}</h2>
+
+Our defined minimum supported versions for WordPress and PHP are set as follows:
+
+- WordPress active major version from the previous year (e.g. when [6.0](https://wordpress.org/news/2022/05/arturo/) was released this would mean [5.7](https://wordpress.org/news/2021/03/esperanza/) is minimum supported version)
```

Comment:
Not sure this explains it better or not, feel free to ignore

```suggestion
- The active major version of WordPress from one year prior (e.g. when [6.0](https://wordpress.org/news/2022/05/arturo/) was released this would mean [5.7](https://wordpress.org/news/2021/03/esperanza/) is minimum supported version)
```